.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: python
 python:
 - '3.6'
-services:
-- postgresql
+addons:
+  postgresql: "9.6"
```

Comment:
Why the change to a specific version of postgresql?

Reply:
I was having some Database issues so I wanted to pin the version we test with to the same one we use in production. Travis defaults to 9.2 but we run 9.6 in prod.